oneview_redfish_toolkit/blueprints/util/response_builder.py

```diff
@@ -77,10 +77,12 @@ def error_404(error):
 
     @staticmethod
     def error_500(error):
-        redfish_error = RedfishError(
-            "InternalError",
-            "The request failed due to an internal service error.  "
-            "The service is still operational.")
+        if hasattr(error, "description"):
+            msg = error.description
+        else:
+            msg = str(error)
```

Comment:
Can we create unit tests raising both types of error?

Reply:
Yes, we can. I will create unit tests for response builder file.